src/VisualStudio/Core/Def/RoslynPackage.cs

```diff
@@ -273,6 +274,7 @@ private void DisposeVisualStudioServices()
             if (_workspace != null)
             {
                 _workspace.Services.GetRequiredService<VisualStudioMetadataReferenceManager>().DisconnectFromVisualStudioNativeServices();
+                _workspace.Services.GetService<IPackageInstallerService>()?.Dispose();
```

Comment:
❕  This is already getting disposed by the MEF catalog.

Reply:
awesome.  thanks.